src/mozanalysis/sizing.py

```diff
@@ -0,0 +1,383 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import attr
+
+from mozanalysis.bq import sanitize_table_name_for_bq
+from mozanalysis.utils import hash_ish
+from mozanalysis.experiment import TimeLimits
+
+
+@attr.s(frozen=False, slots=True)
+class HistoricalTarget:
+    """Query historical data.
+
+    The methods here query data in a way compatible with the following
+    principles, which are important for experiment analysis:
+
+    Example usage (in a colab notebook):
+
+        from google.colab import auth
+        auth.authenticate_user()
+        print('Authenticated')
+
+        from mozanalysis.sizing import HistoricalTarget
+        from mozanalysis.bq import BigQueryContext
+        from mozanalysis.metrics.desktop import active_hours, uri_count
+        from mozanalysis.segments.desktop import new_unique_profiles
+
+        bq_context = BigQueryContext(
+            dataset_id='your-dataset-id',  # e.g. mine's mbowerman
+            project_id='moz-fx-data-bq-data-science'  # this is the default anyway
+        )
+
+        ht = HistoricalTarget(
+            experiment_name='my_test_name',
+            start_date='2021-01-01',
+            num_dates_enrollment=8,
+            print_queries=True
+        )
+
+        # Run the query and get the results as a DataFrame
+        res = ht.get_single_window_data(
+            bq_context,
+            metric_list = [
+                active_hours,
+                uri_count
+            ],
+            last_date_full_data='2021-01-30',
+            analysis_start_days=0,
+            analysis_length_days=21,
+            target_list = [
+                new_unique_profiles,
+                new_or_resurrected_v3
```

Comment:
This object [needs to be imported](https://github.com/mozilla/mozanalysis/pull/152/files#diff-9ac2a7afe5f1013fb39513b6c52d0481e5b4210234d30ddc97f490b7aa410f8bR27) along with `new_unique_profiles`

Reply:
Fixed, thanks!